src/tutorial/furthertopics/clock-triggered-tasks.rst

```diff
@@ -0,0 +1,172 @@
+.. _tutorial-cylc-clock-trigger:
+
+Clock Triggered Tasks
+=====================
+
+.. TODO
+
+   After #2423 has been finalised and merged this tutorial should be
+   re-factored / re-written to incorporate the usage of ``cylc-graph``.
+
+In a :term:`datetime cycling` suite the time represented by the
+:term:`cycle points <cycle point>` bear no relation to the real-world time.
+Using clock-triggers we can make tasks wait until their cycle point time before
+running.
+
+Clock-triggering effectively enables us to tether the "cycle time" to the
+"real world time" which we refer to as the :term:`wall-clock time`.
+
+
+Clock Triggering
+----------------
+
+When clock-triggering tasks we can use different
+:ref:`offsets <tutorial-iso8601-durations>` to the cycle time as follows:
+
+.. code-block:: cylc
+
+   clock-trigger = taskname(CYCLE_OFFSET)
+
+.. note::
+
+   Regardless of the offset used, the task still belongs to the cycle from
+   which the offset has been applied.
+
+
+Example
+-------
+
+Our example suite will simulate a clock chiming on the hour.
+
+Within your ``~/cylc-run`` directory create a new directory called
+``clock-trigger``::
+
+   mkdir ~/cylc-run/clock-trigger
+   cd ~/cylc-run/clock-trigger
+
+Paste the following code into a ``suite.rc`` file:
+
+.. code-block:: cylc
+
+   [cylc]
+       UTC mode = True # Ignore DST
+
+   [scheduling]
+       initial cycle point = TODO
+       final cycle point = +P1D # Run for one day
+       [[dependencies]]
+           [[[PT1H]]]
+               graph = bell
+
+   [runtime]
+       [[root]]
+           [[[events]]]
+               mail events = failed
+       [[bell]]
+           env-script = eval $(rose task-env)
```

Comment:
Was searching for rose when I spotted this one... backtracking a few lines up here, I don't see anything mentioning installing Rose... so I assume it is covered elsewhere, or the user knows what is Rose and how to use it. But just in case...

Reply:
The daft thing is I don't think it's actually necessary anyway.

Reply:
BTW: There is also a reference to the `rose tutorial` command which I will need to migrate to Cylc.